src/docs/product/accounts/pricing.mdx

```diff
@@ -40,9 +40,23 @@ The second is the **On-Demand Period**. The On-Demand Period reflects a one mont
 
 ### Billing Controls
 
-#### On-Demand Spending Cap {#on-demand-cap}
+#### On-Demand Budgets {#on-demand-cap}
 
-You can set a maximum monthly on-demand bill amount by setting an on-demand spending cap. Once you reach the spending cap, you will still be able to access Sentry and prior event data. However, any additional events you send will be rejected.
+You can set a maximum monthly on-demand invoice amount by setting an on-demand budget. Once you reach the maximum budget capacity, you will still be able to access Sentry and prior event data. However, any additional events you send will be rejected.
+
+<Note>
+
+Per-category on-demand budgets are only available on plans with [performance monitoring](/product/performance/) features and if you're in the Early Adopter program.
+If you opted out of the Early Adopter program while on the per-category on-demand budget strategy, you will still retain this on-demand budget until you switch to the shared on-demand budget strategy.
+Customers on plans without performance monitoring can only use the shared on-demand budget strategy.
+
+</Note>
```

Comment:
@imatwawana There are changes to how the new on-demand budget feature will be rolled out. We aim to roll this out to customers on the performance plan, and are also on the early adopters program. 

In addition, if they are using the per-category on-demand budget strategy, they'll retain this configuration regardless if they are on the Early Adopters program or not.

I updated this note to reflect that. I'm unsure if the language here is sufficient.

Reply:
When you say they'll retain the configuration, do we mean that they just retain the per-category budget they've already set up? Or could they make changes to their per-category budget? 

And if they switch to shared, I assume we mean that they can't go back to per-category?

Reply:
@imatwawana 

> When you say they'll retain the configuration, do we mean that they just retain the per-category budget they've already set up? Or could they make changes to their per-category budget?

They'll be able to do both.

So if they:

1. enabled early adopters feature in the past,
2. switched to per-category on-demand budget strategy, and
3. opted out of early adopters afterwards,

they remain on the per-category on-demand budget.

Customers on the performance plans may only be able to opt into per-category on-demand budget only if they're opted into the Early Adopter program. 

> And if they switch to shared, I assume we mean that they can't go back to per-category?

They'll be unable to switch back to per-category as long as they're opted out of Early Adopters.